test-packages/e2e-tests/test/request-builder.spec.ts

```diff
@@ -112,11 +112,16 @@ describe('Request builder test', () => {
   it('should update an entity', async () => {
     const created = await createEntity(entityKey);
     const newDate = moment.utc('2020-01-01', 'YYYY-MM-DD');
-    created.dateProperty = newDate;
-    await TestEntity.requestBuilder().update(created).execute(destination);
+    const newEntity = TestEntity.builder()
+      .keyTestEntity(created.keyTestEntity)
+      .stringProperty(null)
+      .dateProperty(newDate)
+      .build();
+    await TestEntity.requestBuilder().update(newEntity).execute(destination);
 
     const queried = await queryEntity(entityKey);
-    expect(queried.dateProperty?.toISOString()).toBe(
+    expect(queried.stringProperty).toBe(null);
```

Comment:
could you also make an assertion that `created.stringProperty = `someValue`. So that we are sure, that the value of the property has been changed from `someValue`->undefined. Perhaps even better to query the object instead of using the `created` variable because the query will return the state from the remote system for sure. The create could return the local object.